docs-src/develop.rst

```diff
@@ -47,7 +47,7 @@ by linking your source tree from python's ``site-packages``.
 
 Finally, run the all the tests::
 
-    python -m unittest discover
+    CI=1 python -m unittest discover
```

Comment:
I gave you bad advice. This is to skip running some tests in test_data_manager.py and test_gk_data.py that require cloud credentials, but unfortunately this will also skip some tests it's better to run. I'll think of a better solution, in the meantime can you revert the 2 changes to this file?